src/tests/integration-test

```diff
@@ -333,13 +333,13 @@ class UDisksTestCase(unittest.TestCase):
         block.call_format_sync(fs_type, options, None)
 
     @classmethod
-    def retry_busy(cls, fn, *args):
+    def retry_busy(cls, fn, *function_args):
```

Comment:
Yes python will shadow `args` so it has a local scope as expected, but in some cases it could be a bug where you wanted to use the global, but stepped on it in the parameter list.  This change was to remove that ambiguity.  I will rename the global scope in main.